resources/airflow.rb

```diff
@@ -1722,6 +1667,60 @@ def self.create_infil_mech_vent_objects(model, runner, building, infil, mech_ven
       infil_program.addLine("Set Qn = #{building.living.ACH * UnitConversions.convert(building.infilvolume, "ft^3", "m^3") / UnitConversions.convert(1.0, "hr", "s")}")
     end
 
+    if mech_vent.type == Constants.VentTypeBalanced and mech_vent.sensible_effectiveness > 0 and mech_vent.whole_house_cfm > 0
+      # ERV/HRV; balanced systems without energy recovery are modeled via infiltration actuated EMS
+
+      balanced_flow_rate = [UnitConversions.convert(mech_vent.whole_house_cfm, "cfm", "m^3/s"), 0.0000001].max
```

Comment:
The if statement above prevents an airflow of zero, so is this really needed?

Reply:
The If is differentiate ERV from balanced system without heat recovery, do you want it to follow the same logic as ERV instead of actuating `Infiltration` object?

Reply:
Sorry, I am referring to the `[foo, , 0.0000001].max` part of the code here.